### DISCUSSION TITLE
Allow default secondary_info for entities card for all entities

### DISCUSSION THREAD

**Discussion by dshokouhi** [home-assistant/frontend#11288] [Other feature requests]
## The request

A lot of the time when I add entities I want `secondary_info: last-changed` however if I have like 20-30 entities in one card it can take some time to set them up or even convert them.  Just a tiny thing thats all.

## The alternatives

Just to add secondary info to each and every entity

## Additional information

Just to make life easier, some may have an automation card with only automations and want to show last-triggered.  Can be a few more use cases for this too.

**Comment by rodpayne** [comment]
I am also interested in this feature, but does this pull request only address Hui Entities Card?

**Comment by iantrich** [comment]
@rodpayne Yes. I can't think of anywhere else it would be relevant

**Comment by Mariusthvdb** [comment]
bump....
I figured WTH, why not ;-) https://community.home-assistant.io/t/wth-why-is-there-still-no-card-global-secondary-info-last-changed/478374/4
